### PR TITLE
Add `ThatAre[Not]Sealed` to `TypeSelector.cs`

### PR DESCRIPTION
**Pull request by 94sedighi**
## IMPORTANT 

* [x] The code complies with the [Coding Guidelines for C#](https://www.csharpcodingguidelines.com/).
* [x] The changes are covered by unit tests which follow the Arrange-Act-Assert syntax and the naming conventions such as is used [in these tests](../tree/develop/Tests/FluentAssertions.Equivalency.Specs/MemberMatchingSpecs.cs#L51-L430).
* [x] If the PR adds a feature or fixes a bug, please update [the release notes](../tree/develop/docs/_pages/releases.md) with a functional description that explains what the change means to consumers of this library, which are published on the [website](https://fluentassertions.com/releases).
* [x] If the PR changes the public API the changes needs to be included by running [AcceptApiChanges.ps1](../tree/develop/AcceptApiChanges.ps1) or [AcceptApiChanges.sh](../tree/develop/AcceptApiChanges.sh).

### CODE REVIEW

File: Src/FluentAssertions/Types/TypeSelector.cs [hunk 1]
```diff
@@ -183,6 +183,26 @@ public TypeSelector ThatAreNotClasses()
         return this;
     }
 
+    /// <summary>
+    /// Determines wether the type is sealed
```
Comment (dennisdoomen) [inline Src/FluentAssertions/Types/TypeSelector.cs:187]: 🔧 It doesn't determine where a type is sealed, but returns the types that are sealed.

File: Src/FluentAssertions/Types/TypeSelector.cs [hunk 2]
```diff
@@ -183,6 +183,26 @@ public TypeSelector ThatAreNotClasses()
...
+    /// </summary>
+    /// <returns></returns>
+    public TypeSelector ThatAreSealed()
+    {
+        types = types.Where(t => t.IsSealed).ToList();
+        return this;
+    }
+
+    /// <summary>
+    /// Determines wheter the type is not sealed
```
Comment (dennisdoomen) [inline Src/FluentAssertions/Types/TypeSelector.cs:197]: 🔧 Same comment as the other one.

**Review by dennisdoomen — CHANGES_REQUESTED** [review]

**Review by dennisdoomen — APPROVED** [review]

**Comment by dennisdoomen** [comment]
Needs a rebase

**Review by jnyrup — APPROVED** [review]

**Comment by 94sedighi** [comment]
> Needs a rebase
working on int

**Pull request merged**